oak_attestation_verification/src/verifier.rs

```diff
@@ -120,11 +128,35 @@ fn verify_oak_restricted_kernel(
 }
 
 fn verify_oak_containers(
+    now_utc_millis: i64,
     _evidence: &Evidence,
```

Comment:
For my education, what do the leading underscores do?

Reply:
This marks an unused function argument. Flagged by rustc at warning level.